code/game/gamemodes/game_mode.dm

```diff
@@ -541,3 +546,25 @@ proc/get_nt_opposed()
 			to_chat(usr, "[ticker.mode.extended_round_description]")
 	else
 		to_chat(usr, "<i>Shhhh</i>. It's a secret.")
+
+//Handlers for people dying / being revived.
+/datum/game_mode/proc/on_crew_death(mob/living/carbon/human/H)
+	if(!H?.client)
+		return FALSE
+	for(var/X in dead_players) //Hacky, yes. But more consistent than locate() in whatever.
```

Comment:
ckeys aren't the right thing to use, a person can respawn and play more than one character during a round. Other things can take control of a body in some cases

Minds or Real Names or Crew records, ideally minds